From e40b8307dc9fdbae3830dd1c54f6b51de7bfc731 Mon Sep 17 00:00:00 2001
From: kaidoho <kho237115@gmail.com>
Date: Sun, 23 Feb 2020 19:12:01 +0100
Subject: [PATCH] fix build issue

---
 scripts/build/cc/gcc.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/scripts/build/cc/gcc.sh b/scripts/build/cc/gcc.sh
index 6085289d..ad4885da 100644
--- a/scripts/build/cc/gcc.sh
+++ b/scripts/build/cc/gcc.sh
@@ -1191,6 +1191,8 @@ do_gcc_backend() {
         if [ "${CT_THREADS}" = "win32" ]; then
             extra_config+=("--enable-threads=win32")
             extra_config+=("--disable-win32-registry")
+        elif [ "${CT_THREADS}" = "rtems" ]; then
+            extra_config+=("--enable-threads=rtems")
         else
             extra_config+=("--enable-threads=posix")
         fi
-- 
2.17.1

