From ed8803c95ad379742ccc0858620f8dcc97b70ca6 Mon Sep 17 00:00:00 2001
From: Markus Moessner <Markus.Moessner@sick.de>
Date: Tue, 25 Feb 2020 12:26:57 +0100
Subject: [PATCH] Fix build compiler

---
 scripts/build/cc/gcc.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/build/cc/gcc.sh b/scripts/build/cc/gcc.sh
index b464e22..eee3f4b 100644
--- a/scripts/build/cc/gcc.sh
+++ b/scripts/build/cc/gcc.sh
@@ -790,7 +790,7 @@ do_cc_for_build() {
     elif [ "${CT_RTEMS}" = "y" ]; then
         # In the tests I've done, bare-metal was not impacted by the
         # lack of such a compiler, but better safe than sorry...
-        build_final_opts+=( "mode=static" )
+        build_final_opts+=( "mode=rtems" )
         build_final_opts+=( "build_libgcc=yes" )
         build_final_opts+=( "build_libstdcxx=yes" )
         build_final_opts+=( "build_libgfortran=yes" )
@@ -951,7 +951,7 @@ do_cc_libstdcxx_nano()
             if [ "${CT_STATIC_TOOLCHAIN}" = "y" ]; then
                 final_opts+=( "build_staticlinked=yes" )
             fi
-            final_backend=do_gcc_core_backend
+            final_backend=do_gcc_backend
         else
             final_backend=do_gcc_backend
         fi
-- 
2.7.4

