From 7c349df45781001497f49614fe525030a3fc8790 Mon Sep 17 00:00:00 2001
From: Markus Moessner <Markus.Moessner@sick.de>
Date: Tue, 25 Feb 2020 13:09:30 +0100
Subject: [PATCH] fix

---
 scripts/build/cc/gcc.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/build/cc/gcc.sh b/scripts/build/cc/gcc.sh
index eee3f4b..45340ef 100644
--- a/scripts/build/cc/gcc.sh
+++ b/scripts/build/cc/gcc.sh
@@ -797,7 +797,7 @@ do_cc_for_build() {
         if [ "${CT_STATIC_TOOLCHAIN}" = "y" ]; then
             build_final_opts+=( "build_staticlinked=yes" )
         fi
-        build_final_backend=do_gcc_core_backend
+        build_final_backend=do_gcc_backend
     else
         build_final_backend=do_gcc_backend
     fi
@@ -951,7 +951,7 @@ do_cc_libstdcxx_nano()
             if [ "${CT_STATIC_TOOLCHAIN}" = "y" ]; then
                 final_opts+=( "build_staticlinked=yes" )
             fi
-            final_backend=do_gcc_backend
+            final_backend=do_gcc_core_backend
         else
             final_backend=do_gcc_backend
         fi
-- 
2.7.4

